packages/application-extension/src/index.tsx

```diff
@@ -371,20 +374,7 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
       });
 
       commands.addCommand(CommandIDs.toggleSidebarWidget, {
-        label: args =>
-          args === undefined ||
-          args.side === undefined ||
-          args.index === undefined
-            ? trans.__('Toggle Sidebar Element')
-            : args.side === 'right'
-            ? trans.__(
-                'Toggle Element %1 in Right Sidebar',
-                parseInt(args.index as string, 10) + 1
-              )
-            : trans.__(
-                'Toggle Element %1 in Left Sidebar',
-                parseInt(args.index as string, 10) + 1
-              ),
+        label: trans.__('Toggle Sidebar Element'),
```

Comment:
The previous code here needs to be restored. It is critical to rendering shortcuts properly in the settings, otherwise users do not know which toggle they are modifying.